src/services/loggingexport/loggingexportService.js

```diff
@@ -0,0 +1,172 @@
+/**
+ * Simple log export service example which captures messages of specifed categories and levels
+ * and transmits them to a remote API in batch, once a certin batch size or timeout is reached.
+ * 
+ * The service will come up as soon as possible during finsemble startup - but will inevitably
+ * not be able to capture so of the earliest log messages.
+ * 
+ * Look for //TODO: comments for where to customize the service to your needs
```

Comment:
Would it make sense to import a JS file with the necessary customizations?

Reply:
Could do, but thats building an adapter (and fiddly) and really that should be loaded by the central logger so that we're only listening in one place AND we capture all messages (logger is up before loggerexportService). Core team know this and have put a few early thoughts into adapters, but nothing we can use right now.